<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="8" passed="5" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-08T17:41:10 IST" name="Default suite" finished-at="2023-02-08T17:43:13 IST" duration-ms="123073">
    <groups>
    </groups>
    <test started-at="2023-02-08T17:41:10 IST" name="Default test" finished-at="2023-02-08T17:43:13 IST" duration-ms="123073">
      <class name="TestCase.TestAddToCart">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestCase.TestAddToCart@1fa268de]" started-at="2023-02-08T17:41:10 IST" name="setup" finished-at="2023-02-08T17:41:11 IST" duration-ms="1656" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Test_Add_To_Cart()[pri:0, instance:TestCase.TestAddToCart@1fa268de]" started-at="2023-02-08T17:41:11 IST" name="Test_Add_To_Cart" finished-at="2023-02-08T17:41:25 IST" duration-ms="14061" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test_Add_To_Cart -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestCase.TestAddToCart@1fa268de]" started-at="2023-02-08T17:41:25 IST" name="tearDown" finished-at="2023-02-08T17:41:26 IST" duration-ms="195" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCase.TestAddToCart -->
      <class name="TestCase.TestValidCartDetails">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestCase.TestValidCartDetails@36902638]" started-at="2023-02-08T17:41:26 IST" name="setup" finished-at="2023-02-08T17:41:27 IST" duration-ms="1080" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Test_Valid_CartDetails()[pri:0, instance:TestCase.TestValidCartDetails@36902638]" started-at="2023-02-08T17:41:27 IST" name="Test_Valid_CartDetails" finished-at="2023-02-08T17:41:38 IST" duration-ms="10784" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test_Valid_CartDetails -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestCase.TestValidCartDetails@36902638]" started-at="2023-02-08T17:41:38 IST" name="tearDown" finished-at="2023-02-08T17:41:38 IST" duration-ms="182" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCase.TestValidCartDetails -->
      <class name="TestCase.VerifyClearCart">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestCase.VerifyClearCart@8f4ea7c]" started-at="2023-02-08T17:41:50 IST" name="setup" finished-at="2023-02-08T17:41:51 IST" duration-ms="1030" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Verify_Clear_Cart()[pri:0, instance:TestCase.VerifyClearCart@8f4ea7c]" started-at="2023-02-08T17:41:51 IST" name="Verify_Clear_Cart" finished-at="2023-02-08T17:42:08 IST" duration-ms="17053" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Clear_Cart -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestCase.VerifyClearCart@8f4ea7c]" started-at="2023-02-08T17:42:08 IST" name="tearDown" finished-at="2023-02-08T17:42:08 IST" duration-ms="208" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCase.VerifyClearCart -->
      <class name="TestCase.TestSameProductName">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestCase.TestSameProductName@223d2c72]" started-at="2023-02-08T17:41:38 IST" name="setup" finished-at="2023-02-08T17:41:38 IST" duration-ms="706" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Test_Same_ProductName()[pri:0, instance:TestCase.TestSameProductName@223d2c72]" started-at="2023-02-08T17:41:38 IST" name="Test_Same_ProductName" finished-at="2023-02-08T17:41:49 IST" duration-ms="10896" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test_Same_ProductName -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestCase.TestSameProductName@223d2c72]" started-at="2023-02-08T17:41:49 IST" name="tearDown" finished-at="2023-02-08T17:41:50 IST" duration-ms="217" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCase.TestSameProductName -->
      <class name="TestCase.InvalidTestProductSearch">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestCase.InvalidTestProductSearch@436813f3]" started-at="2023-02-08T17:42:08 IST" name="setup" finished-at="2023-02-08T17:42:09 IST" duration-ms="792" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Invalid_Test_ProductSearch()[pri:0, instance:TestCase.InvalidTestProductSearch@436813f3]" started-at="2023-02-08T17:42:09 IST" name="Invalid_Test_ProductSearch" finished-at="2023-02-08T17:42:28 IST" duration-ms="19546" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected ["Automation"] but found [Test]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected ["Automation"] but found [Test]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at TestCase.InvalidTestProductSearch.Invalid_Test_ProductSearch(InvalidTestProductSearch.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Invalid_Test_ProductSearch -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestCase.InvalidTestProductSearch@436813f3]" started-at="2023-02-08T17:42:28 IST" name="tearDown" finished-at="2023-02-08T17:42:28 IST" duration-ms="190" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCase.InvalidTestProductSearch -->
      <class name="TestCase.TestInvalidCartDetails">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestCase.TestInvalidCartDetails@74fe5c40]" started-at="2023-02-08T17:42:28 IST" name="setup" finished-at="2023-02-08T17:42:29 IST" duration-ms="1089" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Test_Invalid_CartDetails()[pri:0, instance:TestCase.TestInvalidCartDetails@74fe5c40]" started-at="2023-02-08T17:42:29 IST" name="Test_Invalid_CartDetails" finished-at="2023-02-08T17:42:43 IST" duration-ms="13512" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Automate It with Zapier: Boost your business productivity using effective workflow automation techniques] but found [Test Automation Learning]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Automate It with Zapier: Boost your business productivity using effective workflow automation techniques] but found [Test Automation Learning]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at TestCase.TestInvalidCartDetails.Test_Invalid_CartDetails(TestInvalidCartDetails.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test_Invalid_CartDetails -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestCase.TestInvalidCartDetails@74fe5c40]" started-at="2023-02-08T17:42:43 IST" name="tearDown" finished-at="2023-02-08T17:42:43 IST" duration-ms="266" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCase.TestInvalidCartDetails -->
      <class name="TestCase.TestDifferentProductName">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestCase.TestDifferentProductName@3febb011]" started-at="2023-02-08T17:42:43 IST" name="setup" finished-at="2023-02-08T17:42:44 IST" duration-ms="1245" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Test_Different_ProductName()[pri:0, instance:TestCase.TestDifferentProductName@3febb011]" started-at="2023-02-08T17:42:44 IST" name="Test_Different_ProductName" finished-at="2023-02-08T17:42:58 IST" duration-ms="13673" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Automate It with Zapier: Boost your business productivity using effective workflow automation techniques] but found [Test Automation Learning]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Automate It with Zapier: Boost your business productivity using effective workflow automation techniques] but found [Test Automation Learning]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at TestCase.TestDifferentProductName.Test_Different_ProductName(TestDifferentProductName.java:41)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Test_Different_ProductName -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestCase.TestDifferentProductName@3febb011]" started-at="2023-02-08T17:42:58 IST" name="tearDown" finished-at="2023-02-08T17:42:58 IST" duration-ms="173" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCase.TestDifferentProductName -->
      <class name="TestCase.ValidTestProductSearch">
        <test-method is-config="true" signature="setup()[pri:0, instance:TestCase.ValidTestProductSearch@10e31a9a]" started-at="2023-02-08T17:42:58 IST" name="setup" finished-at="2023-02-08T17:42:59 IST" duration-ms="913" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Valid_Test_ProductSearch()[pri:0, instance:TestCase.ValidTestProductSearch@10e31a9a]" started-at="2023-02-08T17:42:59 IST" name="Valid_Test_ProductSearch" finished-at="2023-02-08T17:43:13 IST" duration-ms="13334" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Valid_Test_ProductSearch -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:TestCase.ValidTestProductSearch@10e31a9a]" started-at="2023-02-08T17:43:13 IST" name="tearDown" finished-at="2023-02-08T17:43:13 IST" duration-ms="205" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- TestCase.ValidTestProductSearch -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
